.fixtures.yml

```diff
@@ -0,0 +1,4 @@
+fixtures:
+  forge_modules:
```

Comment:
instead of pulling in the forge modules, can you please use the github repos?

```yaml
fixtures:
  repositories:
    stdlib: https://github.com/puppetlabs/puppetlabs-stdlib.git
```